src/src.pro

```diff
@@ -142,8 +142,9 @@ win32 {
 }
 mac {
     RC_FILE = macapp.icns
-    INCLUDEPATH += -I/usr/local/include \
-                   -I/usr/local/opt/sqlite/include
+    QT+= macextras
+    INCLUDEPATH += /usr/local/include \
```

Comment:
Without this patch you will get a '-I-I' in the compiler command. That's why the SQLCipher build broke.